sssd_test_framework/utils/tools.py

```diff
@@ -563,6 +621,19 @@ def group(self, name: str | int) -> GroupEntry | None:
         """
         return self.__exec(GroupEntry, "group", name)
 
+    def initgroups(self, name: str) -> InitgroupsEntry:
+        """
+        Call ``getent initgroups $name``
+
+        If ``name`` does not exist, list of returned object is empty.
```

Comment:
If \``name`` does not exist, group list is empty. This is standard behavior of \``getent initgroups``